build.json

```diff
@@ -8,9 +8,10 @@
     ],
     "RequiredMonoVersion": "6.6.0",
     "DownloadURL": "https://roslynomnisharp.blob.core.windows.net/ext",
-    "MonoRuntimeMacOS": "mono.macOS-6.12.0.90.zip",
-    "MonoRuntimeLinux32": "mono.linux-x86-6.12.0.90.zip",
-    "MonoRuntimeLinux64": "mono.linux-x86_64-6.12.0.90.zip",
+    "MonoRuntimeMacOS": "mono.macOS-6.12.0.122.zip",
+    "MonoRuntimeLinuxX86": "mono.linux-x86-6.12.0.122.zip",
+    "MonoRuntimeLinuxX64": "mono.linux-x86_64-6.12.0.122.zip",
+    "MonoRuntimeLinuxArm64": "mono.linux-x86_64-6.12.0.122.zip",
```

Comment:
is this correct?

Reply:
Thanks, I should have pushed from my linux vm where I made these changes in stead of trying to retype them on mac.